assets/css/style.css

```diff
@@ -0,0 +1,125 @@
+* {
```

Comment:
can you please add this to ::before, ::after too.

Reply:
```
*, *::before, *::after {
    box-sizing: border-box;
}
```